scripts/soltest.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+REPO_ROOT="$(dirname "$0")"/..
+DEBUG_PREFIX=
+BOOST_OPTIONS=
+SOLTEST_OPTIONS=
+CURRENT_OPTIONS=BOOST_OPTIONS
+
+while [ $# -gt 0 ]
+do
+	case "$1" in
+		--debug)
+			shift
+			DEBUG_PREFIX="gdb $1"
+			;;
+		--)
```

Comment:
I would like to avoid that, it is really annoying to type.

Reply:
The thing is that the boost test options also have long versions, e.g. I like to use "--show-progress".

Furthermore some options take an argument, i.e. they are two arguments to the script in total, and if the script decides based on the specific option whether to pass it before or after --, then it has to know whether this is the case for all supported options - I don't like the idea of randomly selecting a subset of options to support in the script or to have to adjust the script for any change in such options...

Reply:
Of course, I could collect all arguments not starting with a "-" and pass them before, resp. after "--" together with the preceding option... one would still loose long options for boost test options. Do you think that would be preferable to having to type "--"?

Reply:
My idea was to just add some known and often used boost options. If you need other options, you cannot use this shortcut script.